content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- dims
+- 44past4
+title: System Logs
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+System component logs record things happening in cluster, which can be very useful for debugging.
+You can configure log verbosity to see more or less detail.
+Logs can be as coarse-grained as showing errors within a component, or as fine-grained as showing step-by-step traces of events (like HTTP access logs, pod state changes, controller actions, or scheduler decisions).
+
+<!-- body -->
+
+## Klog
+
+Kubernetes components use the [klog](https://github.com/kubernetes/klog) logging library.
+[klog](https://github.com/kubernetes/klog) configures logging via command line flags exposed by components.
+You can find their list in [command-line-tools-reference]. Example of klog format:
+
+```
+I1025 00:15:15.525108       1 httplog.go:79] GET /api/v1/namespaces/kube-system/pods/metrics-server-v0.3.1-57c75779f-9p8wg: (1.512ms) 200 [pod_nanny/v0.0.0 (linux/amd64) kubernetes/$Format 10.56.1.19:51756]
+```
+
+[command line tools reference]: /docs/reference/command-line-tools-reference
+[klog]: https://github.com/kubernetes/klog
+
+### Structured Logging
+
+{{<warning>}}
+Migration to structured log messages is an ongoing process. Not all log messages are structured in this version. When parsing log files, you must also handle unstructured log messages.
+
+Structured logging is an alpha feature. Log formatting and value serialization are subject to change.
+{{< /warning>}}
+
+Structured logging is a effort to introduce a uniform structure in log messages allowing for easy extraction of information, making logs easier and cheaper to store and process.
+New message format is backward compatible and enabled by default.
+
+Format of structured logs:
+```
+<klog header> "<message>" <key1>="<value1>" <key2>="<value2>" ...
+```
+
+Example:
+```
+I1025 00:15:15.525108       1 controller_utils.go:116] "Pod status updated" pod="kube-system/kubedns" status="ready"
+```
+
+
+### JSON log format
+
+The `--logging-format=json` flag changes format of logs from components to JSON format.
+
+{{<warning >}}
+JSON output does not support many standard klog flags. The [command line tools reference] lists unsupported klog flags.
+
+Not all logs are guaranteed to be written in JSON format (for example, during process start). If you intend to parse logs, make sure you can handle logs in non-JSON formats.
+
+Structured logging is an alpha feature. Field names and JSON serialization are subject to change.
```

Comment:
Is it appropriate to track this with a feature-state shortcode?

Reply:
Can you explain what it means and link some example how to achieve this?

Reply:
nit: See line 36 in the warning. This is repeated:
`Structured logging is an alpha feature. `
By structured logging, do you mean JSON, --logging-format=json?

Reply:
fixed

Reply:
@serathius the shortcode I mentioned is https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state